include/tvm/relay/executor.h

```diff
@@ -159,9 +159,6 @@ class Executor : public ObjectRef {
  */
 class ExecutorRegEntry {
  public:
-  /*! \brief Set name of the Executor to be the same as registry if it is empty */
-  inline ExecutorRegEntry& set_name();
```

Comment:
Thanks! BTW why the compiler warns this method?

Reply:
here is
```
error: assigning field to itself [-Werror,-Wself-assign-field]
    name = name;
```

Reply:
Ah i see. It's probably a typo?

Reply:
Guess that's some codes forget to be cleaned by authors. Also `RegisterOrGet(const String& name)` in current attr registry mechanism automatically set entry's name already. It should be not necessary to add a zero param `set_name()`.